specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-03-01/managedClusters.json

```diff
@@ -4398,6 +4441,66 @@
           "description": "The type of the extended location."
         }
       }
+    },
+    "OSOptionProperty": {
+      "properties": {
+        "os-type": {
+          "type": "string",
+          "description": "OS type."
+        },
+        "enable-fips-image": {
+          "type": "boolean",
+          "description": "Whether FIPS image is enabled."
+        }
+      },
+      "required": [
+        "os-type",
+        "enable-fips-image"
+      ],
+      "description": "OS option property."
+    },
+    "OSOptionPropertyList": {
+      "properties": {
+        "osOptionPropertyList": {
```

Comment:
Is dash needed on the resource path and property names? I am seeing inconsistencies here and above with - between `os` and `xxx`. I am double checking on naming standards, but at least should be consistent. I suggest remove `-`.

Reply:
this is intended, we want the key to be in the abc-xyz format. The server side model follows the same naming pattern: 

https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=%2Fresourceprovider%2Fserver%2Fmicrosoft.com%2Fcontainerservice%2Fdatamodel%2Fv20210301%2Ftypes.go&version=GBmaster&line=747&lineEnd=748&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents